src/PrestaShopBundle/Controller/Admin/Improve/International/CountryController.php

```diff
@@ -201,6 +235,10 @@ protected function getErrorMessages(Exception $e): array
                 'Country contains invalid field values',
                 'Admin.International.Feature'
             ),
+            DeleteCountryException::class => $this->trans(
+                'Country cannot be deleted',
```

Comment:
```suggestion
                'Country cannot be deleted.',
```

Reply:
there are few more error messages, should i add dot on all of them? 
"Country contains invalid field values"
"Failed to update country"
"This country does not exist"

Reply:
Hello, yes. All error messages should have a final stop. :-) I'll make the suggestion for you